source/iterators.tex

```diff
@@ -1969,6 +1969,7 @@
 \item \tcode{to_address(a) == addressof(*a)},
 \item \tcode{to_address(b) == to_address(a) + D(b - a)},
 \item \tcode{to_address(c) == to_address(a) + D(c - a)},
+\item \tcode{address(I\{\})} is well-defined,
```

Comment:
This should be `to_address` not `address`

Reply:
```suggestion
\item \tcode{to_address(I\{\})} is well-defined,
```

Reply:
Done, thanks!